src/main/java/com/johnstarich/moviematcher/app/MovieMatcherApplication.java

```diff
@@ -146,26 +129,19 @@ public void usersService() {
 			Map<String, Object> userFields = userFieldsOpt.get();
 			String movieId = (String) userFields.get("id");
 			double rating = (double) userFields.get("rating");
-			System.out.println(movieId);
-			System.out.println(rating);
 
 			Optional<Movie> m =  new Movie(new ObjectId(movieId)).load();
 
-			// may need to check if movie is present
+			if(m.isPresent()) {
+				User u = session.get().user;
+				Optional<Rating> r = Rating.loadRatingByUser(u.id, m.get().id);
 
-			User u  = session.get().user;
-			List<Rating> ratings = Rating.loadRatingsByUser(u.id);
+				if(r.isPresent())
+					r.get().delete();
```

Comment:
it was to maintain the idea that user only has one rating on a movie at any time. so if one exist for the movie then i deleted it and created a new one with the updated rating.